test_scripts/API/GetWayPoints/001_GetWayPoints_success_flow.lua

```diff
@@ -0,0 +1,90 @@
+-- Use case:
+-- Item: Use Case 1: Main Flow
+--
+-- Requirement summary:
+-- [GetWayPoints] As a mobile app I want to send a request to get the details of the destination 
+-- and waypoints set on the system so that I can get last mile connectivity.
+--
+-- Description:
+-- In case:
+-- 1) mobile application sends valid and allowed by Policies GetWayPoints_request to SDL
+-- SDL must:
+-- 1) Transfer GetWayPoints_request to HMI
+-- 2) Respond with <resultCode> received from HMI to mobile application
+-- 3) Provide the requested parameters at the same order as received from HMI 
+--    to mobile application (in case of successfull response)
+
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common_get_way_points = require('test_scripts/API/GetWayPoints/common_get_way_points')
+
+local response ={}
+  response["wayPoints"] =
+  {{
+
+      coordinate =
+      {
+        latitudeDegrees = 1.1,
+        longitudeDegrees = 1.1
+      },
+      locationName = "Hotel",
+      addressLines =
+      {
+        "Hotel Bora",
+        "Hotel 5 stars"
+      },
+      locationDescription = "VIP Hotel",
+      phoneNumber = "Phone39300434",
+      locationImage =
+      {
+        value ="icon.png",
+        imageType ="DYNAMIC",
+      },
+      searchAddress =
+      {
+        countryName = "countryName",
+        countryCode = "countryCode",
+        postalCode = "postalCode",
+        administrativeArea = "administrativeArea",
+        subAdministrativeArea = "subAdministrativeArea",
+        locality = "locality",
+        subLocality = "subLocality",
+        thoroughfare = "thoroughfare",
+        subThoroughfare = "subThoroughfare"
+      }
+  } }
+
+
+--[[ Local Functions ]]
+local function get_way_points(params, self)
+    local cid = self.mobileSession1:SendRPC("GetWayPoints", params)
+
+    local deviceID = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+    response.appID = self.applications[config.application1.registerAppInterfaceParams.appName]
+    EXPECT_HMICALL("Navigation.GetWayPoints", params)
+    :Do(function(_,data)
+        --hmi side: sending Navigation.GetWayPoints response
+        self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", response)
+    end)
+
+    self.mobileSession1:ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common_get_way_points.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common_get_way_points.start)
+runner.Step("RAI, PTU", common_get_way_points.rai_ptu)
+runner.Step("Activate App", common_get_way_points.activate_app)
+
+runner.Title("Test")
+runner.Step("GetWayPoints, wayPointType\"ALL\"", get_way_points, {{
+    wayPointType = "ALL"
```

Comment:
@LitvinenkoIra According to arguments for get_way_points() function ```params``` argument is a table with parameters for "GetWayPoints" request
But currently in this invocation instead of table with parameters for RPC a ```{
    wayPointType = "ALL"
  }``` will be sent